charts/gke-operator-crd/templates/crds.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Are we going to backport this to 2.5.x, since they both use the same line right now ? It could get us into a weird state where some of this may not be compatible with the support k8s versions in 2.5.

Reply:
I don't think we wanted to backport this to 2.5, but it looks like `/v1` was introduced in k8s 1.16 https://kubernetes.io/docs/reference/using-api/deprecation-guide/#customresourcedefinition-v122 and 2.5 is supported for >= 1.17 https://rancher.com/support-maintenance-terms/all-supported-versions/rancher-v2.5.9/ so I think this would be safe to keep in the same line

Reply:
What tag is this commit going into ? We already released 1.1.1 and are adding that to 2.5 for QA purposes: https://github.com/rancher/system-charts/blob/dev-v2.5/charts/rancher-gke-operator/1.0.400/values.yaml#L7

Is the CRD going to v1 the only big change getting imported here ? With the 1.22 changes coming up we may need to split anyway and undo our idea of keeping 2.5 and 2.6 in line.

Reply:
The CRD is the only change. I would guess the next tag for gke-operator should be 1.1.2 for 2.6.x but maybe we should have a conversation about that because I'm not sure.

Reply:
I had the same question about tag. I think 1.1.2 is OK because any upgrade issues to 2.6.x would be based on chart version.

Reply:
> I think 1.1.2 is OK because any upgrade issues to 2.6.x would be based on chart version.
Can you clarify what you mean by this ? 

I think we should chat. Need to clearly define the goal of the older branch and if we should keep it around, or if we need to branch a 3rd time. Sounds like we may be able to get away with 1.1.1 everywhere.